src/main/java/bio/terra/cli/serialization/userfacing/UFConfig.java

```diff
@@ -67,6 +74,7 @@ public void print() {
             + fileLoggingLevel);
     OUT.println();
     OUT.println("[server] server = " + serverName);
```

Comment:
Maybe skip the whole line if `workspaceId == null`?  I feel like this would look strange as written.

Reply:
How about `[workspace] workspace = (unset)` instead? Agreed about it looking weird with just a dangling `=`. But I think it's helpful to have all the possible config properties listed out.

Reply:
Works for me, thanks.